### PR TITLE
Feat/618 various updates

### PR DESCRIPTION
**Pull request by morgnism**
# Overview

* Simplified subcategories
* Adds "State of Node/State of the Web" to meetups communities list
* Appends the list of general conferences used on the angular/vue/react site for communities (see original issue for the list of conferences spreadsheet)

## Type of change

- [x] Content addition
- [ ] Bug fix
- [ ] Behavior change

## Checklist

- [x] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [x] I have searched all existing content and verified my additions are novel
      and unique
- [x] The content was added to the end of the appropriate data list
- [x] All required content fields are accurately populated
- [x] All items are tagged with all relevant tags

### CODE REVIEW

**Comment by jdwilkin4** [comment]
When I running it locally, I noticed a broken image for Feathers.
Screenshot 2023-01-17 at 2 54 22 PM

We could throw in that fix into this PR since this is a general cleanup PR

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
I think it looks good. 
It was just those two broken image links that needed replacing 👍

File: packages/site/src/data/nodejs/libraries.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const libraries: Library[] = [
 			"A progressive Node.js framework for building efficient, reliable and scalable server-side applications.",
 		image:
 			"https://d33wubrfki0l68.cloudfront.net/e937e774cbbe23635999615ad5d7732decad182a/26072/logo-small.ede75a6b.svg",
-		tags: [LibraryTag.WEB_SERVERS, LibraryTag.API_LIBRARIES],
+		tags: [LibraryTag.FRAMEWORKS, LibraryTag.FRAMEWORKS],
```
Comment (lindakatcodes) [inline packages/site/src/data/nodejs/libraries.ts:33]: This creates 2 tags for frameworks - we should only need the one I'd think?

File: packages/site/src/data/nodejs/libraries.ts [hunk 2]
```diff
@@ -81,7 +81,7 @@ export const libraries: Library[] = [
 			"Sails.js is a web framework that makes it easy to build custom, enterprise-grade Node.js apps. It is designed to resemble the MVC architecture from frameworks like Ruby on Rails, but with support for the more modern, data-oriented style of web app & API development. It's especially good for building realtime features like chat.",
 		image:
 			"https://camo.githubusercontent.com/7809ea02014a21791f76317fa04c16d460d14b9ad4db2f6166e48aa4e329d628/687474703a2f2f62616c64657264617368792e6769746875622e696f2f7361696c732f696d616765732f6c6f676f2e706e67",
-		tags: [LibraryTag.API_LIBRARIES, LibraryTag.FRAMEWORKS],
+		tags: [LibraryTag.FRAMEWORKS, LibraryTag.FRAMEWORKS],
```
Comment (lindakatcodes) [inline packages/site/src/data/nodejs/libraries.ts:84]: Duplicate tag

File: packages/site/src/data/nodejs/libraries.ts [hunk 3]
```diff
@@ -157,7 +157,7 @@ export const libraries: Library[] = [
 			"Redwood is the full-stack web framework designed to help you grow from side project to startup.",
 		image:
 			"https://d33wubrfki0l68.cloudfront.net/72b0d56596a981835c18946d6c4f8a968b08e694/82254/images/logo.svg",
-		tags: [LibraryTag.CLI, LibraryTag.UTILITIES, LibraryTag.WEB_SERVERS],
+		tags: [LibraryTag.FRAMEWORKS, LibraryTag.UTILITIES, LibraryTag.FRAMEWORKS],
```
Comment (lindakatcodes) [inline packages/site/src/data/nodejs/libraries.ts:160]: Duplicate tag for framework

File: packages/system/src/models/library-tag.ts [hunk 4]
```diff
@@ -7,6 +7,7 @@ export enum LibraryTag {
 	TESTING = "testing",
 	UTILITIES = "utilities",
 	WEB_SERVERS = "web servers",
+	SERVERLESS = "serverless",
```
Comment (lindakatcodes) [inline packages/system/src/models/library-tag.ts:10]: Shouldn't this be "serverless framework", per the ticket?

**Review by lindakatcodes — CHANGES_REQUESTED** [review]
Found a few things, but I'd think they should be fairly quick to address:
- Some duplicated tags in the libraries file, I pointed out which ones - just needs the duplicate removed
- Correction in the library-tag model I think should be changed per the ticket

Everything else looks good! And I see State of Node is already there, so that's good. :)

**Review by jdwilkin4 — APPROVED** [review]
Looks good 👍

File: packages/system/src/models/library-tag.ts (hunk 4 above)
Comment (lindakatcodes) [reply packages/system/src/models/library-tag.ts:10]: So do we need this addition here at all then? If it's new from this PR and we're not using it - should it be removed completely?

**Review by lindakatcodes — APPROVED** [review]

**Pull request merged**